src/file.cc

```diff
@@ -278,9 +278,28 @@ void write_buffer_to_fd(Buffer& buffer, int fd)
     }
 }
 
-void write_buffer_to_file(Buffer& buffer, StringView filename)
+void write_buffer_to_file(Buffer& buffer, StringView filename, bool force)
 {
-    int fd = open(filename.zstr(), O_CREAT | O_WRONLY | O_TRUNC, 0644);
+    int fd;
```

Comment:
I thought it would look cleaner to move this variable at the top, since it's used several times. Up to you.